src/kirby/components/list/list.component.tns.html

```diff
@@ -5,10 +5,14 @@
     </FlexboxLayout>
   </ng-container>
   <RadListView [groupingFunction]="getSectionName" [items]="items" row="{{ rowNumberForListView(headerTemplate) }}">
-    <ng-template let-item="item" tkListItemTemplate>
```

Comment:
Don't we need the `tkListItemTemplate` to have cell reuse?

I am not sure here as my nativescript knowledge is limited.

Reply:
I am not sure either. I have added it.